tests/test_grouping.py

```diff
@@ -498,7 +498,10 @@ def test_comparison_with_strings(operator):
     assert p.tokens[0].right.ttype == T.String.Single
 
 
-def test_like_and_ilike_comparison():
+@pytest.mark.parametrize('operator', (
```

Comment:
Adding some extra coverage to all possible operators we are touching here as they are all a little different (LIKE since it acts differently for table creation, ilike because it is a default keyword, and rlike because it is not)